src/schema/schema.graphql

```diff
@@ -289,6 +289,12 @@ enum ResetPasswordStatus {
 	suspended
 }
 
+enum PrefClientType {
+	advanced,
```

Comment:
this is confusing, we have two options only, right?

Reply:
we are getting three options from API response.